source/shared/cpp/ObjectModel/ChoiceSetInput.cpp

```diff
@@ -9,8 +9,7 @@
 using namespace AdaptiveCards;
 
 ChoiceSetInput::ChoiceSetInput() :
-    BaseInputElement(CardElementType::ChoiceSetInput), m_wrap(false), m_isMultiSelect(false),
-    m_choiceSetStyle(ChoiceSetStyle::Compact)
+    BaseInputElement(CardElementType::ChoiceSetInput), m_wrap(false), m_isMultiSelect(false), m_choiceSetStyle(ChoiceSetStyle::Compact)
```

Comment:
nit: inconsitency; why does this one make a longer line? the previous changes in this PR took the longer ones to multiple lines.

Reply:
I have no idea. they both should have passed the formatting and verification.